### DISCUSSION TITLE
Support for Mermaid

### DISCUSSION THREAD

**Discussion by amduenas07** [skoropadas/ng-doc#164] [Q&A] [answered]
Is there any possibility to render a diagram with Mermaid?

**Comment by skoropadas** [comment, marked as answer]
Actually, I was thinking about it but wasn't sure if it's gonna be used by someone. I'll add it, could you transform the discussion into an issue?

**Reply by amduenas07** [reply]
Ready! Thank you for answering so quickly :)
